rllib/core/models/configs.py

```diff
@@ -246,15 +246,191 @@ def build(self, framework: str = "torch") -> Model:
         self._validate(framework=framework)
 
         if framework == "torch":
-            from ray.rllib.core.models.torch.mlp import TorchFreeLogStdMLPHead
+            from ray.rllib.core.models.torch.heads import TorchFreeLogStdMLPHead
 
             return TorchFreeLogStdMLPHead(self)
         else:
-            from ray.rllib.core.models.tf.mlp import TfFreeLogStdMLPHead
+            from ray.rllib.core.models.tf.heads import TfFreeLogStdMLPHead
 
             return TfFreeLogStdMLPHead(self)
 
 
+@ExperimentalAPI
+@dataclass
+class CNNTransposeHeadConfig(ModelConfig):
+    """Configuration for a convolutional transpose head (decoder) network.
+
+    The configured Model transforms 1D-observations into an image space.
+    The stack of layers is composed of an initial Dense layer, followed by a sequence
+    of Conv2DTranspose layers.
+    `input_dims` describes the shape of the (1D) input tensor,
+    `initial_image_dims` describes the input into the first Conv2DTranspose
```

Comment:
If one does not read the full docstring, I think initial_image_dims can be mistaken to be the input dims of the image that is to be reconstructed, so the output dims of the transpose2d.
I think "first_transpose_layer_input_dims" would avoid this.